### ISSUE TITLE
Error deleting persistent volumes when operators are used

### ISSUE THREAD

**Issue by bashofmann** [kubermatic/kubermatic#3696]
A customer cluster is stuck in deletion when also deleting PVCs with this error message:

```
E0627 12:02:34.356739       1 cluster_controller.go:203] Failed to reconcile cluster v27sc64dbn: failed to clean up PV using workloads from user cluster: [kind "Prometheus" is unknown kind "Prometheus" is unknown]
{"level":"error","ts":1561636954.3568354,"logger":"kubebuilder.controller","msg":"Reconciler error","controller":"kubermatic_kubernetes_controller","request":"/v27sc64dbn","error":"failed to clean up PV using workloads from user cluster: [kind \"Prometheus\" is unknown kind \"Prometheus\" is unknown]","stacktrace":"github.com/kubermatic/kubermatic/api/vendor/github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/kubermatic/kubermatic/api/vendor/github.com/go-logr/zapr/zapr.go:128\ngithub.com/kubermatic/kubermatic/api/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/src/github.com/kubermatic/kubermatic/api/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:217\ngithub.com/kubermatic/kubermatic/api/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/go/src/github.com/kubermatic/kubermatic/api/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:158\ngithub.com/kubermatic/kubermatic/api/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/src/github.com/kubermatic/kubermatic/api/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\ngithub.com/kubermatic/kubermatic/api/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/kubermatic/kubermatic/api/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:134\ngithub.com/kubermatic/kubermatic/api/vendor/k8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/kubermatic/kubermatic/api/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88"}
```

The cluster was using the prometheus-operator to deploy prometheus, though very likely this will fail with any other operator as well that creates a PV.

We are currently running 2.10 with this PR already cherry-picked: https://github.com/kubermatic/kubermatic/pull/3608

**Comment by alvaroaleman** [comment]
This can be done by adding a `ValidatingWebhookConfiguration` on all clusters that were deleted for `CREATE` of PVs and PVCs and a failurePolicy of `fail` and no associated webhook. That will make creating new PVs and PVCs impossible.

**Comment by bashofmann** [comment]
@alvaroaleman I just tried it with the fixes that were also merged into the 2.11 branch and I still get this error:

```
E0724 12:38:37.323154       1 cluster_controller.go:204] Failed to reconcile cluster 86snz6d84v: failed to clean up PV using workloads from user cluster: [kind "MysqlCluster" is unknown]
```

**Comment by bashofmann** [comment]
I can confirm that it works now :).
